data-sources/litecoind/litecoin.conf

```diff
@@ -3,6 +3,5 @@ server=1
 rpcuser=rpcuser
 rpcpassword=rpcpassword
 txindex=1
-daemon=1
```

Comment:
It's important that litecoin isn't running in daemon mode since docker requires that the main application runs in the foreground.